nx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "namedInputs": {
+    "default": ["{projectRoot}/src/**/*"]
+  },
+  "workspaceLayout": {
+    "appsDir": "apps",
+    "libsDir": "packages"
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["default"],
+      "outputs": ["{projectRoot}/dist"],
+      "cache": true
+    },
+    "lint": {
+      "dependsOn": ["build"],
```

Comment:
for typescript this is needed for sure, since repack includes dev-server and utilizies some types from there, and it's probably the same case for typescript-eslint tbh, or eslint plugin import (won't be able to resolve some imports if the package is not built)

Reply:
Now that I think about it, maybe we could point to sources directly so that it's possible to lint in dev without building 🤔

Reply:
managed to find a way to do this 🎉